roles/internal/apix/tasks/config.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+- name: Template settings
+  template:
+    src: apix.cfg.j2
+    dest: "{{ apix_karaf_etc_dir }}/{{ item.pid }}.cfg"
```

Comment:
Oh, I see.  It's here.  Either drop the .cfg's in the pids or drop it here and put them all the pid entries at `roles/internal/apix/defaults/main.yml`